src/conv-core/convcore.C

```diff
@@ -1632,12 +1637,22 @@ void CsdBeginIdle(void)
 #else
   CpvAccess(cmiMyPeIdle) = 1;
 #endif // CMK_SMP
-  CcdRaiseCondition(CcdPROCESSOR_BEGIN_IDLE) ;
+  double curWallTime = CcdRaiseCondition(CcdPROCESSOR_BEGIN_IDLE) ;
```

Comment:
Should we change the function signature or should we just have a separate CmiWallTimer call?
If we are going with the former, the Converse programming manual should also be updated.

Reply:
Oh yeah, good point. I can update the converse programming manual. 

I think it's better to go ahead with CcdRaiseCondition returning the current wall time since it can save a couple of additional timer calls. (and esp since it was returning void before this).